database/insert_test_data.sql

```diff
@@ -6,7 +6,7 @@ truncate smartsaver.goal restart identity cascade;
 truncate smartsaver.tag restart identity cascade;
 truncate smartsaver.transaction_tag restart identity cascade;
 
--- SET CLIENT_ENCODING TO 'UTF8';
+SET CLIENT_ENCODING TO 'UTF8';
```

Comment:
As I understand nothing has changed?

Reply:
In database structure nothing.
However, now lithuanian letters are saved correctly (`ą` is saved in database as `ą`)